awx/main/models/ha.py

```diff
@@ -32,7 +32,7 @@
 
 
 def validate_queuename(v):
-    # celery and kombu don't play nice with unicode in queue names
+    # kombu doesn't play nice with unicode in queue names
```

Comment:
This probably isn't really true anymore; at a later date, we could probably investigate lifting this restriction.